coldfront/plugins/qumulo/tests/utils/mock_data.py

```diff
@@ -228,3 +230,173 @@ def set_allocation_attributes(
                 allocation=allocation,
                 value=form_data.get(allocation_attribute_name),
             )
+
+
```

Comment:
@harterjwustl Thanks for organizing the data using these methods. These mock data can be reusable in future test cases (I hope).